openapi_python_client/templates/property_templates/union_property.pyi

```diff
@@ -9,6 +9,17 @@ def _parse_{{ property.python_name }}(data: {{ property.get_type_string(json=Tru
     if isinstance(data, Unset):
         return data
     {% endif %}
+    {% if property.discriminator_property != None %}
```

Comment:
This results in a chain that looks like this for RegisteredEntitiesList, which has a discriminator:

```
                discriminator_value: str = data.get("entityType")
                if discriminator_value is not None:
                    if discriminator_value == "dna_sequence":
                        entities_item = DnaSequence.from_dict(data)

                        return entities_item
                    if discriminator_value == "custom_entity":
                        entities_item = CustomEntity.from_dict(data)

                        return entities_item
                    if discriminator_value == "aa_sequence":
                        entities_item = AaSequence.from_dict(data)

                        return entities_item
                    if discriminator_value == "mixture":
                        entities_item = Mixture.from_dict(data)

                        return entities_item
                    if discriminator_value == "dna_oligo":
                        entities_item = DnaOligo.from_dict(data)

                        return entities_item
                    if discriminator_value == "rna_oligo":
                        entities_item = RnaOligo.from_dict(data)

                        return entities_item
```